Modules/Daq/test/testQcDaq.cxx

```diff
@@ -36,7 +37,7 @@ BOOST_AUTO_TEST_CASE(instantiate_task)
   task.startOfCycle();
   auto producer = AliceO2::DataSampling::DataBlockProducer(false, 1024);
   DataSetReference dataSet = producer.getDataSet();
-  task.monitorDataBlock(dataSet);
+//  task.monitorDataBlock(dataSet);// TODO
```

Comment:
The problem is that we still do not have a feature to test the algorithms in separation. The only way would be to create actual workflow executable, send some test data to tested data processor, and exit with 0 if the results are correct. This method is actually used in the main O2 repo.